tools/runners/run-test-suite-test262.sh

```diff
@@ -52,23 +52,30 @@ fi
 rm -rf "${PATH_TO_TEST262}/test/suite/bestPractice"
 rm -rf "${PATH_TO_TEST262}/test/suite/intl402"
 
-# TODO: Enable these tests after daylight saving calculation is fixed.
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T1.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T2.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T3.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T4.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T5.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T6.js"
+echo "Starting test262 testing for ${ENGINE}. Running test262 may take several minutes."
 
-echo "Starting test262 testing for ${ENGINE}. Running test262 may take a several minutes."
+function progress_monitor() {
+  NUM_LINES_EXPECTED=11558
+  NUM_LINES_GOTTEN=0
+  (>&2 echo)
+  while read line; do
+    FRAC=$((NUM_LINES_GOTTEN * 100 / NUM_LINES_EXPECTED))
+    (>&2 echo -ne "\rEstimated ${FRAC}% complete (${NUM_LINES_GOTTEN}/${NUM_LINES_EXPECTED})...")
+    echo "$line"
+    NUM_LINES_GOTTEN=$((NUM_LINES_GOTTEN+1))
+  done
+  (>&2 echo)
+  (>&2 echo)
+}
```

Comment:
Now, this is something that I'd like to see in a different PR. This is a feature/improvement that has nothing to do with the goals of this current PR. This change could work no matter what port API is in place.